### PR TITLE
table layouts, etc.

### PR DESCRIPTION
**Pull request by katyhuff**
Made some grammatical changes, updated titlepage (whoops, I guess I pointed you to the old version), cleaned up the tables, gave it a number (UIUC-ARFC-19-03).

### CODE REVIEW

**Comment by robfairh** [comment]
@katyhuff can you add here the .pdf?

**Comment by katyhuff** [comment]
[uiuc-arfc-2019-03.pdf](https://github.com/arfc/ddca_numerical_exp/files/3443750/uiuc-arfc-2019-03.pdf)

File: docs/uiuc-arfc-2019-03/uiuc-arfc-2019-03.tex [hunk 1]
```diff
@@ -887,21 +884,22 @@ \subsection{Buffer Size}
...
-	& Cumulative {[}GW{]}                                                        & 60.0 & 87.3 & 52.9 & 68.3           & 93.3 \\ \hline
-	\multirow{2}{*}{2000}                       & \begin{tabular}[c]{@{}l@{}}No. of time steps\\ of undersupply\end{tabular} & 20   & 18   & 1    & 20             & 2    \\ \cline{2-7} 
-	& Cumulative {[}GW{]}                                                        & 20.0 & 7.7  & 0.3  & 11.0           & 60.3 \\ \hline
-	\multirow{2}{*}{4000} & \begin{tabular}[c]{@{}l@{}}No. of time steps\\ of undersupply\end{tabular} & 0    & 0    & 0    & 0              & 1    \\ \cline{2-7} 
-	& Cumulative {[}GW{]}                                                        & 0    & 0    & 0    & 0              & 60.  \\ \hline
-	\end{tabular}
-\end{table*}
+        \begin{tabularx}{\textwidth}{llcRRRR}
+                \hline
+        Buffer      &                      & MA   & ARMA  & POLY & EXP       & FFT  \\ \hline
```
Comment (robfairh) [inline docs/uiuc-arfc-2019-03/uiuc-arfc-2019-03.tex]:
```suggestion
        Buffer      &                      & MA   & ARMA  & POLY & EXP       & FFT  \\
```

**Comment by katyhuff** [comment]
@robfairh extra line resolved, your commit accepted.

**Comment by katyhuff** [comment]
Thanks @robfairh

**Pull request merged**